integration/env/vars.bat

```diff
@@ -21,3 +21,5 @@ if not defined DPL_ROOT for /f "delims=" %%F in ("%VARSDIR%..") do set "DPL_ROOT
 
 set "CPATH=%DPL_ROOT%\windows\include;%CPATH%"
 set "PKG_CONFIG_PATH=%DPL_ROOT%\lib\pkgconfig;%PKG_CONFIG_PATH%"
+
+set set "CMAKE_PREFIX_PATH=%DPL_ROOT%\lib\cmake\WindowsIntelLLVM;CMAKE_PREFIX_PATH=%DPL_ROOT%\lib\cmake\oneDPL;%CMAKE_PREFIX_PATH%"
```

Comment:
I think it is enough to add `CMAKE_PREFIX_PATH=%DPL_ROOT%\lib\cmake\oneDPL` if we are going to put `Windows IntelLLVMApply.cmake` and `WindowsIntelLLVMConfig.cmake` there.

Reply:
My intention was to separate it to its own subfolder in the installed version, but I don't have a strong preference to do this, I am OK with placing it in `%DPL_ROOT%\lib\cmake\oneDPL`.

Reply:
You commented it above:
> In the long term, this does not have much to do with oneDPL.

And now I understand a reason for that. Let's leave it as is.

Reply:
I see that  `WindowsIntelLLVM` files have been renamed to `oneDPLWindowsIntelLLVM`, so now it looks slightly better set `CMAKE_PREFIX_PATH` to `%DPL_ROOT%\lib\cmake\oneDPL`.

Reply:
I don't have a strong preference one way or the other here.  For now its perhaps best to have it in the same folder, and name it with `oneDPL` prefix.